pkg/splunk/enterprise/names.go

```diff
@@ -250,3 +271,38 @@ func GetSplunkImage(specImage string) string {
 func GetPortName(port string, protocol string) string {
 	return fmt.Sprintf(portNameTemplateStr, protocol, port)
 }
+
+// GetProbeConfigMapName uses a template to name a Kubernetes ConfigMap for a Liveness/Readiness Probe.
+func GetProbeConfigMapName(identifier string) string {
+	return fmt.Sprintf(probeConfigMapTemplateStr, identifier)
+}
+
+// GetReadinessScriptLocation return the location of readiness script
+var GetReadinessScriptLocation = func() string {
```

Comment:
Is there any reason why we are using function pointers instead of constants here?

Reply:
In unit test cases we needed to ScriptLocation. Defining as var helps us with that.

Reply:
Can't we use a constant in names.go and use it directly? Since there is no `fmt.Sprintf` here we could use `readinessScriptLocation` directly?

Reply:
Then we would have no way of allocating new `readinessScriptLocation` in the unit test which will cause failures due to different relative path. 
This is a simpler approach to the problem. The other solution would be to copy the scripts to relative location before test start and cleaning up after the test finishes. 
Now we can just modify the location like [here](https://github.com/splunk/splunk-operator/blob/CSPL-2018/pkg/splunk/enterprise/clustermaster_test.go#L47)